src/Sulu/Bundle/AdminBundle/Resources/js/stores/ResourceStore/ResourceStore.js

```diff
@@ -279,6 +294,31 @@ export default class ResourceStore {
         );
     }
 
+    @action setMultipleRecursive(data: Object, parentPath: string = '') {
```

Comment:
This is necessary, because `setMultiple` sets only the root properties and thus `blocks` is replaced by `blocks`. 

But to set the `diff` from the `calculateDifference` method, we have to recursively navigate through the individual blocks and replace only single blocks instead of all of them.

I implemented a new method for this, because the performance of the simple `setMultiple` is obviously better and the recursive method is only required during the data merge of this pr.